content/docs/views.md

```diff
@@ -0,0 +1,102 @@
+* **Show All Characters**: When selected, this is effectively both **Show Whitespace and TAB** and **Show End of Line** at the same time.
+
+When the **Show Indent Guide** entry is selected, a [colored](#global-styles "Style Configurator > Global Styles > Indent Guideline Style") dotted vertical line `⸽` will show where each of the tab stops are located if there are enough tabs or spaces at the start of a given line to go beyond a tab stop (as defined by the [per-language Tab settings](#language)).
+
+When the **Show Wrap Symbol** entry is selected and **Word Wrap** is also enabled, there will be a little [colored](#global-styles "Style Configurator > Global Styles > White Space Symbol") arrow `↲` on the far edge of the current line if that line has been wrapped.
+
+## Zoom
+
+The **Zoom** sub-menu will allow you to **Zoom In** (enlarge the text), **Zoom Out** (shrink the text), or **Restore Default Zoom** (which will change the text size back to default settings).  These actions all act only on the current View, so if you have two Views visible, only one will be affected.
+
+## Move / Clone
+
+The **Move/Clone Current Document** submenu will allow you to affect where the current document is viewed.
+
+If you **Move to Other View**, the current file will be moved from one View to the other.  If there is currently only one View visible, this command will make the second View visible -- it may be to the right, below, to the left, or above the original View, depending on where the second view was last placed.  To change the arrangement of the Views, right-click on the dotted bar between the two Views, and **Rotate Right** or **Rotate Left**, which will change the orientation from side-to-side to above-and-below.  You can also accomplish this **Move to Other View** by dragging the tab from one View's tab-bar to the other, or right clicking on the tab and selecting the **Move to Other View** from that menu.
+
+If you **Clone to Other View**, the current file will be visible in both Views at the same time.
+
+If the current file is currently not modified (so no unsaved changes), you can also either **Move to New Instance** or **Clone to New Instance**, which will move the active file from this instance to a new instance of Notepad++, or make this file open in both the current Notepad++ and a new instance of Notepad++.  This will work to create a new instance of Notepad++ even if the [**Settings > Preferences > Multi-Instance**](../preferences/#multi-instance) is set to "mono-instance".
+
+## Tab
+
+The **Tab** sub-menu will allow you to navigate through the various open tabs of the active View.
+
+The numbered entries will activate the _N_th tab. The **Next tab** and **Previous tab** actions will activate the next or previous tab in the list of tabs (wrapping).

```

Comment:
Should this _N_th be an underscore? This is rendered as is now (in GitHub markdown preview).

Reply:
fixed